server/validation.es6

```diff
@@ -132,19 +128,18 @@ validate.validateNoncommercial = (obj) => {
 
   errorArr = validateSchema(validationObj);
 
-  if(obj.dateTimeRange && !util.validateDateTime(obj.dateTimeRange.startDateTime)) {
+  if (obj.dateTimeRange && !util.validateDateTime(obj.dateTimeRange.startDateTime)) {
     errorArr.push('pattern-dateTimeRange.startDateTime');
   }
 
-  if(obj.dateTimeRange && !util.validateDateTime(obj.dateTimeRange.endDateTime)) {
+  if (obj.dateTimeRange && !util.validateDateTime(obj.dateTimeRange.endDateTime)) {
     errorArr.push('pattern-dateTimeRange.endDateTime');
   }
 
   return errorArr;
 };
 
-validate.validateTempOutfitter = (obj) => {
-
+validate.validateTempOutfitter = obj => {
```

Comment:
My personal preference is that even with one argument, we should use parens to make it clear that this is a function arg.